### ISSUE TITLE
After screen turn off touch stops wroking

### ISSUE THREAD

**Issue by GregoryGHarding** [MatthewBooth/JFLTE-GPE#20]
Once screen turns off, when you wake up phone sometimes the touch screen is unresponsive and you have to lock/unlock screen a few times until touch screen will respond. Running most recent update on alucard. Used hulk before and never had this issue. Is anyone experiencing this since update?

**Comment by MatthewBooth** [comment]
This should be fixed in latest version.

**Comment by defyance** [comment]
issue still occuring after update

**Comment by MatthewBooth** [comment]
No, it doesn't. All three options have been tested throughly. You have to clean install if you were facing the issue.

**Comment by GregoryGHarding** [comment]
how do i clean install then. the options were removed from the installer

**Comment by MatthewBooth** [comment]
Same way you would for any other ROM :) Do it in recovery. Wipe system and data (you can save internal_sd).

**Comment by GregoryGHarding** [comment]
im dumb. thanks did factory reset installing again now

**Comment by airwavesIRE** [comment]
Factory reset still left me with some issue's,  wiping system and data then a factory reset fixed them.

**Comment by GregoryGHarding** [comment]
ok. noted. will restart precess. my poor phone lol

**Comment by GregoryGHarding** [comment]
i think i found the problem, its up to you if you want to duplicate, but i've changed my DPI to 400, seems after that there's touchscreen issues upon wakeup

**Comment by MatthewBooth** [comment]
I'm not really sure how a DPI setting would affect the kernel.

**Comment by MatthewBooth** [comment]
This will be less of an issue in the next update, as it will use a stock GPE kernel with some modifications by me.

**Comment by GregoryGHarding** [comment]
alucard not working out?

**Comment by MatthewBooth** [comment]
It works great for me, on AOSP, CM and GPE, but everyone is complain. So, I'm adding a few features to the stock kernel, and calling it a day. Easier anyway (and it at least boots quicker, haha).
